app/upgrades/v20/upgrades.go

```diff
@@ -0,0 +1,144 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package v20
+
+import (
+	"fmt"
+	"math/big"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/evmos/evmos/v19/contracts"
+	evmostypes "github.com/evmos/evmos/v19/types"
+	erc20keeper "github.com/evmos/evmos/v19/x/erc20/keeper"
+	erc20types "github.com/evmos/evmos/v19/x/erc20/types"
+	evmoscore "github.com/evmos/evmos/v19/x/evm/core/core"
+	"github.com/evmos/evmos/v19/x/evm/core/vm"
+	evmkeeper "github.com/evmos/evmos/v19/x/evm/keeper"
+	"github.com/evmos/evmos/v19/x/evm/statedb"
+)
+
+// CreateUpgradeHandler creates an SDK upgrade handler for v19
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	bk bankkeeper.Keeper,
+	erc20k erc20keeper.Keeper,
+	ek *evmkeeper.Keeper,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		logger := ctx.Logger().With("upgrade", UpgradeName)
+		logger.Info("Running migrations...")
+		if err := AddCodeToERC20Extensions(ctx, erc20k, ek, bk); err == nil {
+			return nil, err
+		}
+
+		return mm.RunMigrations(ctx, configurator, vm)
+	}
+}
+
+// AddCodeToERC20Extensions adds code and code hash to the ERC20 precompiles with the EVM.
+func AddCodeToERC20Extensions(ctx sdk.Context,
+	erc20Keeper erc20keeper.Keeper,
+	evmKeeper *evmkeeper.Keeper,
+	bankKeeper bankkeeper.Keeper,
+) error {
+	// need the interpreter to get the code after running the constructor
+	interpreter, err := getInterpreter(ctx, evmKeeper)
```

Comment:
why? you can use `GetCode` from the EVM Keeper

Reply:
We'll have to check if it's still stored the code with the corresponding codeHash